2021-12-13 19:48:26,877 INFO    MainThread:14424 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 19:48:26,877 INFO    MainThread:14424 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 19:48:26,877 INFO    MainThread:14424 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_194826-12vfedyf\logs\debug.log
2021-12-13 19:48:26,878 INFO    MainThread:14424 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_194826-12vfedyf\logs\debug-internal.log
2021-12-13 19:48:26,878 INFO    MainThread:14424 [wandb_init.py:init():404] calling init triggers
2021-12-13 19:48:26,878 INFO    MainThread:14424 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 19:48:26,879 INFO    MainThread:14424 [wandb_init.py:init():449] starting backend
2021-12-13 19:48:26,879 INFO    MainThread:14424 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 19:48:26,883 INFO    MainThread:14424 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 19:48:26,974 INFO    MainThread:14424 [backend.py:ensure_launched():204] started backend process with pid: 11708
2021-12-13 19:48:26,975 INFO    MainThread:14424 [wandb_init.py:init():458] backend started and connected
2021-12-13 19:48:26,976 INFO    MainThread:14424 [wandb_init.py:init():519] updated telemetry
2021-12-13 19:48:26,998 INFO    MainThread:14424 [wandb_init.py:init():549] communicating current version
2021-12-13 19:48:27,532 INFO    MainThread:14424 [wandb_init.py:init():554] got version response 
2021-12-13 19:48:27,532 INFO    MainThread:14424 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 19:48:27,770 INFO    MainThread:14424 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 19:48:27,873 INFO    MainThread:14424 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 19:48:27,875 INFO    MainThread:14424 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 19:48:27,875 INFO    MainThread:14424 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 19:48:27,877 INFO    MainThread:14424 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 19:48:27,877 INFO    MainThread:14424 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 19:48:27,877 INFO    MainThread:14424 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000025D59776970>>
2021-12-13 19:48:27,877 INFO    MainThread:14424 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 19:48:27,878 INFO    MainThread:14424 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000025D59776970>>
2021-12-13 19:48:27,878 INFO    MainThread:14424 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 19:48:27,879 INFO    MainThread:14424 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 19:48:27,879 INFO    MainThread:14424 [wandb_run.py:_restore():1758] restore
2021-12-13 19:48:28,897 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:48:29,158 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:48:29,801 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2743
}

2021-12-13 19:48:29,916 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2769
}

2021-12-13 19:48:30,027 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2769
}

2021-12-13 19:48:30,132 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:30,241 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:30,350 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:30,460 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:30,569 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:30,919 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}

2021-12-13 19:48:31,455 INFO    MainThread:14424 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2769
  total_bytes: 2769
}
local_info {
}

2021-12-13 19:48:32,510 INFO    MainThread:14424 [wandb_run.py:_append_files():2184] logging synced files
